### DISCUSSION TITLE
Asyncio installation

### DISCUSSION THREAD

**Discussion by Notadumdum** [miguelgrinberg/python-socketio#1229] [Q&A] [answered]
If I want to make use of python socket.io asyncio client version 4.6.1 how will I install it... should I install the regular client and then asyncio library or there is a different python socket.io asyncio client library

**Comment by miguelgrinberg** [comment, marked as answer]
Asyncio comes with Python, does not need to be installed. To learn how to install Python-SocketIO, please consult the documentation. Here is a link to the client installation for version 4.6.1: https://github.com/miguelgrinberg/python-socketio/blob/v4.6.1/docs/client.rst#installation

**Comment by Notadumdum** [comment]
How can I catch all events from the clientside sir.. like a  wildcard or something

**Reply by miguelgrinberg** [reply]
You are using a fairly old version of the client, in that version there is no wildcard event handler.

**Reply by Notadumdum** [reply]
Is there no client version that uses EIO=3 that has a catch all or wildcard event?

**Reply by miguelgrinberg** [reply]
No, this was done more recently.

**Reply by Notadumdum** [reply]
Gracias
